app/models/manageiq/providers/openstack/inventory/collector/target_collection.rb

```diff
@@ -104,19 +105,31 @@ def orchestration_stacks
     []
   end
 
-  def get_orchestration_stack(stack_id, tenant_id)
-    tenant = memoized_get_tenant(tenant_id)
-    safe_get { @os_handle.detect_orchestration_service(tenant.try(:name)).stacks.get(stack_id) }
```

Comment:
@aufi it looks like this wasn't fetching stack `details`, so I wonder if this worked correctly (fetching all the needed data). 
So check if `/v1/{tenant_id}/stacks/{stack_identity}` gives you the detail, or we need to do 2 calls